pxc/pxc57-build.yml

```diff
@@ -102,6 +102,12 @@
 
         find ./target # For testing results and log
 
+        if [ -f /usr/bin/yum ]; then
+            if [[ $(rpm --eval %rhel) -eq 7 ]]; then
+                export PATH="/usr/local/bin:$PATH"
+            fi
+        fi
```

Comment:
why is it needed?

Reply:
Because CentOS 7's `python3-pip` installs awscli in /usr/local/bin which is not in default path.